homeassistant/components/device_tracker/gpslogger.py

```diff
@@ -5,37 +5,32 @@
 https://home-assistant.io/components/device_tracker.gpslogger/
 """
 import asyncio
+from functools import partial
 import logging
-from hmac import compare_digest
 
-from aiohttp.web import Request, HTTPUnauthorized  # NOQA
 import voluptuous as vol
 
-import homeassistant.helpers.config_validation as cv
-from homeassistant.const import (
-    CONF_PASSWORD, HTTP_UNPROCESSABLE_ENTITY
-)
-from homeassistant.components.http import (
-    CONF_API_PASSWORD, HomeAssistantView
-)
+from homeassistant.const import HTTP_UNPROCESSABLE_ENTITY
+from homeassistant.components.http import HomeAssistantView
 # pylint: disable=unused-import
 from homeassistant.components.device_tracker import (  # NOQA
-    DOMAIN, PLATFORM_SCHEMA
-)
+    DOMAIN, PLATFORM_SCHEMA)
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['http']
 
+CONF_MAX_GPS_ACCURACY = 'max_gps_accuracy'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_MAX_GPS_ACCURACY): vol.Coerce(float),
```

Comment:
Removing `CONF_PASSWORD` will break the setup of users who are using this feature.